src/sql/workbench/parts/query/browser/queryActions.ts

```diff
@@ -278,6 +278,10 @@ export class CancelQueryAction extends QueryTaskbarAction {
 
 	public run(): Promise<void> {
 		if (this.isConnected(this.editor)) {
+			if (!this.editor.input) {
+				console.error('editor input was null');
+				return Promise.resolve(null);
```

Comment:
Should just be Promise.resolve() - this method returns Promise\<void>

Reply:
I'm following existing code here to make my changes as minimal as possible: https://github.com/microsoft/azuredatastudio/blob/f4c2e28262f5cf81c811ecb99fbc09718fc7703d/src/sql/workbench/parts/query/browser/queryActions.ts#L219

Reply:
I'd actually suggest fixing that up too then while you're here. What it's doing right now is just wrong and at best is confusing and at worst could cause unintended behavior if someone sees that and expects to get something back.